packages/contracts/test/src/CredbullFixedYieldVaultFactoryTest.t.sol

```diff
@@ -2,47 +2,51 @@
 
 pragma solidity ^0.8.19;
 
-import { Test } from "forge-std/Test.sol";
-import { HelperVaultTest } from "../base/HelperVaultTest.t.sol";
-import { CredbullFixedYieldVaultFactory } from "../../src/factories/CredbullFixedYieldVaultFactory.sol";
-import { DeployVaultFactory } from "../../script/DeployVaultFactory.s.sol";
-import { HelperConfig, NetworkConfig } from "../../script/HelperConfig.s.sol";
-import { CredbullFixedYieldVault } from "../../src/CredbullFixedYieldVault.sol";
 import { IAccessControl } from "@openzeppelin/contracts/access/IAccessControl.sol";
-import { CredbullVaultFactory } from "../../src/factories/CredbullVaultFactory.sol";
-import { CredbullKYCProvider } from "../../src/CredbullKYCProvider.sol";
+
+import { Test } from "forge-std/Test.sol";
+
+import { HelperConfig, NetworkConfig } from "@script/HelperConfig.s.sol";
+import { DeployVaultFactory } from "@script/DeployVaultFactory.s.sol";
+
+import { CredbullFixedYieldVaultFactory } from "@src/CredbullFixedYieldVaultFactory.sol";
+import { CredbullFixedYieldVault } from "@src/CredbullFixedYieldVault.sol";
+import { CredbullKYCProvider } from "@src/CredbullKYCProvider.sol";
+import { VaultFactory } from "@src/factory/VaultFactory.sol";
```

Comment:
i like using the aliasing for clarity, but i think we can do better than @src and @script.

e.g. could we include @credbull in the alias?  especially since we've removed "Credbull" as a prefix from contract names.

Reply:
I actually had it as `@credbull` initially, but then found that `@credbull-test` and `@credbull-script` started getting quite long, which defeated the point, I thought. I will revert to something along those lines.